tools/ports/libpng.py

```diff
@@ -18,7 +18,7 @@ def needed(settings):
 
 
 def get_lib_name(settings):
-  return 'libpng' + ('-mt' if settings.USE_PTHREADS else '') + '.a'
+  return 'libpng' + ('-mt' if settings.USE_PTHREADS or settings.WASM_WORKERS else '') + '.a'
```

Comment:
Is it safe to use the same name for pthreads or wasm workers? I'd guess we need a separate one?

Reply:
In term of the build flags for the ports, they are the same
Those ports are also ignoring the build definition:
- __EMSCRIPTEN_PTHREADS__
- __EMSCRIPTEN_WASM_WORKERS__

Do you think it is still better to make a separated one?

## Copied from emcc.py
```
if settings.USE_PTHREADS or settings.WASM_WORKERS:
settings.SHARED_MEMORY = 1

if settings.USE_PTHREADS and '-pthread' not in newargs:
newargs += ['-pthread']
elif settings.SHARED_MEMORY:
if '-matomics' not in newargs:
  newargs += ['-matomics']
if '-mbulk-memory' not in newargs:
  newargs += ['-mbulk-memory']

build flags:
-matomics
-mbulk-memory 

```